### DISCUSSION TITLE
golanci-lint working locallyg but fails on CI/CD (Typecheck error)

### DISCUSSION THREAD

**Discussion by wesleymotaaraujo** [golangci/golangci-lint-action#608] [Q&A]
Hi!

The golanci-list is working locally, but it's falling on CI/CD (Github actions).

I've started to receive these typecheck errors :

```golang

  Error: repo.DB undefined (type Repo has no field or method DB) (typecheck)
  Error: repo.DB undefined (type Repo has no field or method DB) (typecheck)
  Error: repo.DB undefined (type Repo has no field or method DB) (typecheck)
  Error: "github.com/lib/pq" imported but not used (typecheck)
```
Has anyone faced something similar?

Thanks!

**Comment by bombsimon** [comment]
`typecheck` is not a real linter but a way to represent compilation errors. This indicates that your code does not compile. What result do you get if you run `go build ./...` in CI? Do you use any generated code or similar which may exist locally but not in CI?
